### ISSUE TITLE
Excessive API calls during startup

### ISSUE THREAD

**Issue by acruis** [HubTurbo/HubTurbo#779]
![screen shot 2015-07-11 at 2 28 43 am](https://cloud.githubusercontent.com/assets/8437638/8626046/ef71b052-2774-11e5-999f-1fff509b578e.png)

Made worse with the fact that metadata retrievals aren't tagged with ETags. Currently the API consumes about 50 counts per refresh due to an `updated:12

**Comment by damithc** [comment]
Is this still relevant?

**Comment by acruis** [comment]
Yes. #849 helps reduce the calls to a negligible amount, but there is some sort of race condition going on during start up, which should be solved later.
